build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -11,7 +11,7 @@
   <ItemGroup>
     <PublishOutputExtensions Include="$(ExeExtension);.dll;.pdb;.deps.json;.runtimeconfig.json" />
     <FilesToMove Include="$(BinaryToCorehostifyRelDir)/csc.exe;$(BinaryToCorehostifyRelDir)/MSBuild.exe;" />
-    <BundledTools Include="csc;MSBuild;NuGet.CommandLine.XPlat;dotnet" />
+    <BundledTools Include="csc;MSBuild;NuGet.CommandLine.XPlat;dotnet;vstest.console;testhost;datacollector" />
```

Comment:
Do we need all 3 of these "tools"?  What are the other 2 used for?  I don't see any commands calling into them.